src/functions/errorMessages.ts

```diff
@@ -11,10 +11,19 @@ export type ErrorMessageMessage =
   | "Beginning with http(s):// is a must."
   | "The link is too long."
   | undefined;
+export type ErrorTagState = "assigned" | "empty" | "submit" | "wrong";
+export type ErrorTagMessage =
+  | "Tag already assigned"
+  | ""
```

Comment:
Is an empty string an error message? I think it would make more sense to have an undefined at that moment instead of an empty string.

Reply:
Great, that makes absolute sense